Tests/Indexer/Indexer.php

```diff
@@ -66,6 +66,7 @@ public function setApiSettings($application_id, $api_key, $connection_timeout =
             if ($kernel->getContainer()->getParameter('algolia.get_credentials_from_env')) {
                 $application_id = getenv('ALGOLIA_APPLICATION_ID');
                 $api_key = getenv('ALGOLIA_API_KEY');
+                var_dump($application_id, $api_key); die;
```

Comment:
Travis automatically hide them, but yeah that's not a good way to go ^^ I should have used `strlen` for instance.